Source/Orts.Parsers.Msts/TokenID.cs

```diff
@@ -1718,6 +1718,9 @@ public enum TokenID : uint
         MaxStackedContainers,
         Length,
         Flipped,
+
+        // TSRE specific
+        Ruler
```

Comment:
It's a simple fix, thanks. My only question is whether ruler appears in compressed .w files and as such needs to be assigned a specific number. I think TSRE does not compress .w files, so it's not required, right?

Reply:
I don't think the numbers that you pointed from the TSRE sources have anything to do with compressed .w files. From my understanding of the compressed SBR format, each token is assigned a value, but I'm not familiar with that either. Anyway, .w files produced by TSRE are not compressed, so this is not a problem.